Examples/CargoFactory/AIRTransferFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Examples.Cargo
+{
+    public class AIRTransferFactory : ITransferFactory
```

Comment:
you have air factory, and it's responsibility is create and manage the instance of Air. but you did it in Air class.